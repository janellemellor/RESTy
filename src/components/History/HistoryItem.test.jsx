import React from 'react';
import { shallow } from 'enzyme';
import HistoryItem from './HistoryItem';

describe('History Item Component', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<HistoryItem url='https://github.com/janellemellor' method='GET' />);
    expect(wrapper).toMatchSnapshot();
  });  
});

