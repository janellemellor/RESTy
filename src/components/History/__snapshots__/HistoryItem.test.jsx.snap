// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`History Item Component matches a snapshot 1`] = `
<section>
  <p>
    GET
  </p>
  <p>
    https://github.com/janellemellor
  </p>
</section>
`;
